.github/workflows/ci.yml

```diff
@@ -18,30 +18,38 @@ jobs:
   test:
     needs: check
     if: needs.check.outputs.branch-pr == ''
+    strategy:
+      matrix:
+        runs-on: ["ubuntu-latest"] # can add windows-latest, macos-latest
+        python-version: ["3.11"]
+        include:
+          # Include one that runs in the dev environment
+          - runs-on: "ubuntu-latest"
+            python-version: "dev"
+      fail-fast: false
     uses: ./.github/workflows/_test.yml
     with:
-      python-version: dev
-      runs-on: ubuntu-latest
+      runs-on: ${{ matrix.runs-on }}
+      python-version: ${{ matrix.python-version }}
     secrets:
       CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
 
-  docs:
```

Comment:
Is this intentional?

Reply:
Ah this was a mistake, must have overwritten the whole file instead of just copying a few minor changes